tool/tctl/common/resource_command.go

```diff
@@ -616,6 +616,19 @@ func (rc *ResourceCommand) Delete(client auth.ClientI) (err error) {
 			return trace.Wrap(err)
 		}
 		fmt.Printf("lock %q has been deleted\n", rc.ref.Name)
+	case types.KindDatabaseServer:
+		dbServers, err := client.GetDatabaseServers(ctx, apidefaults.Namespace)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		for _, server := range dbServers {
+			if server.GetName() == rc.ref.Name {
+				if err := client.DeleteDatabaseServer(ctx, apidefaults.Namespace, server.GetHostID(), server.GetName()); err != nil {
+					return trace.Wrap(err)
+				}
+				fmt.Printf("database server (HostID: %s Name: %s) has been deleted\n", server.GetHostID(), server.GetName())
```

Comment:
Nit: I think that "host ID" and the fact that each database agent is represented as a separate object is an internal implementation detail and might confuse the user. I would just print a single "database server %q has been deleted" message once we've removed all matching servers.

Reply:
Right, changed to single message without HostID details.